node/bft/ledger-service/src/ledger.rs

```diff
@@ -352,7 +352,7 @@ impl<N: Network, C: ConsensusStorage<N>> LedgerService<N> for CoreLedgerService<
             metrics::update_block_metrics(block);
         }
 
-        tracing::info!("\n\nAdvanced to block {} at round {} - {}\n", block.height(), block.round(), block.hash());
+        tracing::info!("[{}] {}", block.height(), block.hash());
```

Comment:
While more verbose, the previous message of `Advanced to lock {} at round {}` has been very valuable for debugging and parsing logs.